lib/index.coffee

```diff
@@ -44,7 +44,7 @@ module.exports = (opts) ->
 
       @b = browserify
         entries: @files
-        extensions: ['.js', '.json', '.coffee']
+        extensions: opts.opts.extensions
```

Comment:
This is cool, but can you also default them as they exist so this is not enormously breaking?

```coffee
opts.opts.extensions or ['.js', '.json', '.coffee']
```